create_infotexts.py

```diff
@@ -100,6 +100,22 @@ def load_places_mapping():
 
     return places_dict
 
+def create_commons_filename(metadata, fotonr):
+    """Creates commons filename according to https://phabricator.wikimedia.org/T156612 and ouputs to new_dict
+    :type populated_dict: dictionary
+    """
+
+    if not metadata[fotonr]["Beskrivning"] == "":
+        cleaned_fname = helpers.format_filename(metadata[fotonr]["Beskrivning"], "SMVK-MM-Cypern", metadata[fotonr]["Fotonummer"])
+        #print("Fname using BatchUploadTools: {}".format(cleaned_fname))
+    else:
+        # TODO: fix alternative description according to https://phabricator.wikimedia.org/T156612#3008806 [Issue: https://github.com/mattiasostmar/SMVK-Cypern_2017-01/issues/9]
+        beskr = "Svenska Cypernexpeditionen 1927-1931"
+        cleaned_fname = helpers.format_filename(beskr, "SMVK-MM-Cypern", metadata[fotonr]["Fotonummer"])
+
+    return cleaned_fname + ".tif"
```

Comment:
I don't believe the `.tif` file extension should be added, file endings are handled by PrepUpload